### DISCUSSION TITLE
Featured Radio Stations

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Tauon#437] [Show and tell]
I currently don't have this documented anywhere else, but there is extended support in Tauon for particular radio stations allowing for album art. :radio: :framed_picture: 

These are:

 - Nightwave Plaza
 - Yggdrasil Radio
 - Listen.moe Jpop & Kpop

You can find these in a built-in secret selection of stations by going "Internet Radio" > "Browse" > Hold <kbd>Shift</kbd> key > "Taikos Favorites".

**Comment by Solarunit** [comment]
Looks like a very solid list! :smiley: 
I have added them to the favorites. Maybe I will start to listen to the radio.

![](https://i.imgur.com/52bQe3F.png)
